### PR TITLE
Email verification and mass email sending to subscribers completed

### PR DESCRIPTION
**Pull request by gawsalya**
Email function complete -- now includes a body and subject for the email.
Verification sent to those users who are not SES verified and reports sent to those that are!

### CODE REVIEW

File: report/lambda_function.py [hunk 1]
```diff
@@ -569,10 +597,19 @@ def handler(event, context) -> None:
...
     create_report(game_df, config["DASHBOARD_URL"])
     print("Report created.")
-    send_email(config)
-    print("Email sent.")
+    verification_awaited = []
+    for address in all_emails:
+        try:
+            send_email(config, address)
+            print("Report email sent.")
+        except:
```
Comment (DMcCallion) [inline report/lambda_function.py]: Maybe be more specific here, is there a specific error we'll get if the user's email isn't verified yet?

**Review by DMcCallion — APPROVED** [review]
Looks good, just a minor suggestion

File: report/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 altair
 boto3
+botocore
```
Comment (angelikavilde) [inline report/requirements.txt]: installed with boto3

Comment (angelikavilde) [reply report/requirements.txt]: should not be included

**Review by angelikavilde — APPROVED** [review]

**Pull request merged**